static/js/utils/generateInitAnswersExperienceFrameFunction.js

```diff
@@ -1,8 +1,7 @@
 import { sendToIframe } from '../iframe-common';
 
 /**
- * Returns a function which sends a message to the iframe to initialize Answers
- * with the provided runtimeConfig
+ * Returns a function which updates the runtime config and initializes answers
```

Comment:
Small nit, but could we have a more compact name for this function? It's a bit of a mouth full

Reply:
Due to the refactor which added the AnswersExperienceFrame class, this method is no longer needed